internal/db/schema/migrations/postgres/dev/86_iam.up.sql

```diff
@@ -0,0 +1,60 @@
+begin;
+
+-- add the account_info_auth_method_id which determines which auth_method is
+-- designated as for "account info" in the user's scope.  
+alter table iam_scope
+add column account_info_auth_method_id wt_public_id -- allowed to be null and is mutable of course.
+references auth_method(public_id)
+on update cascade
+on delete cascade;
+
+-- iam_user_acct_info provides account info for users by determining which
+-- auth_method is designated as for "account info" in the user's scope via the
+-- scope's account_info_auth_method_id.  Every sub-type of auth_account must be
+-- added to this view's union.
+create view iam_acct_info as
+select 
+    aa.iam_user_id,
+    oa.subject_id as login_name,
+    oa.full_name as full_name,
+    oa.email as email
+from 	
+    iam_scope s,
+    auth_account aa,
+	auth_oidc_account oa
+where
+    aa.public_id = oa.public_id and 
+    aa.auth_method_id = s.account_info_auth_method_id 
```

Comment:
What will this view return if `s.account_info_auth_method_id` is null?

Reply:
nada, via the inner join.  that's why I did an outer join in iam_user_acct_info